scripts/Muon/GUI/ElementalAnalysis/PeriodicTable/periodic_table_view.py

```diff
@@ -6,7 +6,7 @@
 
 
 class PeriodicTableView(QtGui.QWidget):
-    sig_table_changed = QtCore.pyqtSignal(list)
+    sig_table_changed = QtCore.pyqtSignal(object)
     sig_table_clicked = QtCore.pyqtSignal(
```

Comment:
you could probably make this one an object too

Reply:
The only reason I can think of for changing the signal would be to add more arguments - not sure anything other than the whole class would be useful here

Reply:
but we would not lose anything by changing it

Reply:
I'm happy to change it, was just thinking there would be a clarity trade-off